examples/option_field_ignore_empty.proto

```diff
@@ -27,7 +27,7 @@ message SubmitFeedbackRequest {
   string email = 2 [
     // `ignore_empty` skips validation on a field if it's empty.
     // The email will be not validated if it's empty.
-    (buf.validate.field).ignore_empty = true,
+    (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED,
```

Comment:
The comment above says:
>     // `ignore_empty` skips validation on a field if it's empty.
>     // The email will be not validated if it's empty.

Can we update to

>     // `IGNORE_IF_UNPOPULATED` skips validation on a field if it's unpopulated.
>     // The email will be not validated if it's the empty string.

?

Reply:
Yes, of course. Thanks!